### ISSUE TITLE
`home-manager news` should provide an option to only show "unread" news

### ISSUE THREAD

**Issue by NobbZ** [nix-community/home-manager#1533]
### Issue description

`home-manager news` prints a *huge* lists of "news", most of them are neither relevant for me, or I already read them.

Sometimes `home-manager switch` informs me about a given number of "unread and relevant news items".

Sadly those are not necessarily at the top of the paged output, but I have to sometimes find them somewhere in between.

Therefore it would be nice to have a way to only show *unread* news items.

**Comment by teto** [comment]
I vote for ditching news.nix altogether, its problematic on rebase, introduce some states, and duplicates git log effort.

**Comment by teto** [comment]
@rycee I am williing to put in the work. What do you think of removing news altogether ?

**Comment by sumnerevans** [comment]
I'm for keeping news as a concept, but I wouldn't mind helping to improve it. Maybe even make it automated: every week (or whatever cadence) generate summary of new and changed options and publish something online (possibly with an RSS feed) so that people who care can know when new stuff gets merged?

**Comment by teto** [comment]
I am not against newsletter/changelog, on the contrary. 
There is some level of automation that can be reached with simple conventions as https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-header

I dont recount the number of rebase due to news.nix clashing. And now there is a flake issue: https://github.com/nix-community/home-manager/issues/2033

**Comment by fricklerhandwerk** [comment]
Also in favor of not printing news automatically. I will use `git log` in any case, and having some automation affordances around that would indeed help. Agreed, no point duplicating work or reinventing wheels around version control.

**Comment by teto** [comment]
yet another PR blocked by news.nix: https://github.com/nix-community/home-manager/pull/2984 we could use a wiki instead.
